tools/docker/functions.sh

```diff
@@ -16,6 +16,11 @@ status() {
 	printf '\033[1;35m'"$@\n"'\033[0m'
 }
 
+convert_to_hex(){
+   mac=`echo $@ | tr --delete :`
+   mac="0x${mac}"
+}
```

Comment:
Now after looking at the second commit, I think it would be much simpler to have this function echo the output instead of putting it in `mac` global variable which is pretty ugly.

So the function as I would prefer it would simply be:
```
mac_to_hex() {
    echo "0x$(echo $1 | tr --delete :)"
}
```

So it can be used as follows:
```
mac=$(mac_to_hex 00:11:22:33:44:55)
```
```